### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by AreTillery**
- Move from % formatting to using .format
- Remove time-of-check-time-of-use bug introduced by querying filesystem prior to opening file

### CODE REVIEW

File: attack_json_to_bullets.py [hunk 1]
```diff
@@ -31,15 +31,14 @@ def get_data_from_branch(domain, branch="master"):
     args = parser.parse_args()
     # Load custom layer JSON
     # First, try for a local file
-    if path.exists(args.jsonfile) and path.isfile(args.jsonfile):
+    try:
         with open(args.jsonfile) as f:
             custom_layer = json.load(f)
-    # If not a local file, try URL
-    else:
+    except (FileNotFoundError, IsADirectoryError):
```
Comment (scytheadam) [inline attack_json_to_bullets.py]: I think this needs to be, `except (FileNotFoundError, IsADirectoryError, OSError):` to work when querying remote URL (the URL is invalid argument).

Comment (AreTillery) [reply attack_json_to_bullets.py]: I don't follow. If you supply a URL, then the file open will fail with `FileNotFoundError`, which will then get caught by the except clause, which will try to open it using `requests`

Comment (AreTillery) [reply attack_json_to_bullets.py]: image

Comment (scytheadam) [reply attack_json_to_bullets.py]: Just tested without catching the `OSError`:

Screen Shot 2020-08-14 at 12 44 05 PM

Comment (AreTillery) [reply attack_json_to_bullets.py]: It looks like the behaviour differs on Windows. Alright, easy fix.

Comment (AreTillery) [reply attack_json_to_bullets.py]: Resolved in the newest commit

**Pull request merged**